administrator/components/com_associations/Documentation.md

```diff
@@ -0,0 +1,38 @@
+#### Introduction
+
+to do.
+
+#### Associations Component (com_associations) Requirements
+
+1. Component should have a `ROOT/components/<com_componentname>/helpers/association.php` Example [here](https://github.com/joomla/joomla-cms/blob/staging/components/com_content/helpers/association.php)
+
+2. `ROOT/component/<component_name>/helpers/association.php` needs to load `route.php`:
```

Comment:
above you put `<com_componentname>` here you put `<component_name>`. please normalize all expressions.